### ISSUE TITLE
RStudio view displays NA as 0(integer64 type column in dataframe)

### ISSUE THREAD

**Issue by mstkolf** [rstudio/rstudio#6869]
### System details

    RStudio Edition : Desktop
    RStudio Version : Version 1.2.5042
    OS Version      : macOS Catalina(version 10.15.4)
    R Version       : R version 3.5.2 (2018-12-20)

### Steps to reproduce the problem

When you print the data in the console, these values will be NA correctly (even in int64 column).

``` r
library(tibble)

(
tbl <- tibble(
  col_dbl = c(NA, 2, 3, 4, 5),
  col_int64 = bit64::as.integer64(c(NA, 2, 3, 4, 5))
  )
)
#> # A tibble: 5 x 2
#>   col_dbl col_int64
#>     <dbl> <int64>  
#> 1      NA NA       
#> 2       2  2       
#> 3       3  3       
#> 4       4  4       
#> 5       5  5
```

<sup>Created on 2020-05-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

But, if you click on a dataset to see the preview(or `View(tbl)`), it shows first row value in col_int64 as 0, see the attached screenshot.
<img width="643" alt="RStudio view displays NA as 0(integer64 type column in dataframe)" src="https://user-images.githubusercontent.com/35163523/81893519-cf3c2c80-95e8-11ea-9b77-cad3b9bce3f7.png">

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issMue.

**Comment by kevinushey** [comment]
Note: would need to handle specially here.

https://github.com/rstudio/rstudio/blob/master/src/cpp/session/modules/SessionDataViewer.R#L24-L51

**Comment by ronblum** [comment]
@mstkolf Thank you for filing the issue! I am able to reproduce it using RStudio Desktop 1.3.958 with R 4.0.0 and RStudio Server with R 3.6.3. We'll review this issue as we continue development of RStudio.

**Comment by R-Hannibal** [comment]
@mstkolf, thanks again for finding and reporting this issue. 

We have a fix in place in our latest Daily build of the upcoming 1.4 release of RStudio (available [HERE](https://dailies.rstudio.com/)). If you get the chance will you let us know if the issue is resolved for you in that build?

**Comment by kfeinauer** [comment]
Verified fixed on 1.4.944
